include/nil/crypto3/zk/snark/arithmetization/plonk/constraint_system.hpp

```diff
@@ -119,6 +119,36 @@ namespace nil {
                         return _lookup_tables[table_id];
                     }
 
+                    std::size_t sorted_lookup_columns_number() const {
+                        std::size_t result = 0;
+                        if(_lookup_gates.size() == 0){
```

Comment:
Probably no need to check this, it will not speed up anything... Since we have functions lookup_options_num() and lookup_constraints_num() , better just write 

return lookup_constraints_num() + lookup_options_num();